Makefile

```diff
@@ -31,6 +31,22 @@ rpm: $(PACKAGE_NAME).spec tarball
 	rm -r "`pwd`/rpmbuild"
 	rm -r "`pwd`/build"
 
+rpm-nodeps: $(PACKAGE_NAME).spec tarball
+	- rm -r "`pwd`/output"
+	mkdir -p "`pwd`/output"
+	mkdir -p "`pwd`/rpmbuild"
+	/usr/bin/rpmbuild -bb \
+	  --nodeps \
```

Comment:
`rpm` does a dependency check when building?! The man page says "Don't do a dependency check before installing or upgrading a package."

Are you sure we need this?

Reply:
Yup, it checks for builddeps (like the python macros).